vNext/shared/AppInsightsCore/src/JavaScriptSDK/AppInsightsCore.ts

```diff
@@ -145,11 +145,6 @@ export class AppInsightsCore implements IAppInsightsCore {
             }
         });
 
-        // Remove sender channels from main list
-        if (c < this._extensions.length) {
-            this._extensions.splice(c);
-        }
```

Comment:
What was this code meant to do?

Reply:
This code is basically removing ChannelController from the array, I don't see any use of this, _extensions is only used in track method after this and having Channel Controller in there doesn't look to affect any scenario, _extensions is also a public property but is only used in SKU to identify if Properties extension is available

Reply:
Adding this logic back and including check for empty extensions during track